plugins/analytics-module-matomo/package.json

```diff
@@ -12,7 +12,12 @@
   "backstage": {
     "role": "frontend-plugin",
     "supported-versions": "1.28.4",
-    "pluginId": "analytics-module-matomo"
+    "pluginId": "analytics-module-matomo",
+    "pluginPackages": [
+      "@janus-idp/backstage-plugin-analytics",
```

Comment:
this don't depend on / include the matomo backend and common plugins, only the analytics plugin?